sapi/fpm/status.html.in

```diff
@@ -215,7 +215,7 @@
 					doc_rate.disabled = false;
 				} else {
 					delay = parseInt(doc_rate.value);
-					if (!delay || delay < 1) {
+					if (delay < 1) {
```

Comment:
this file has `.in` ext, does some C file needs to be regenerated?

Reply:
If `delay` is `NaN` this is incorrect.

Reply:
Above there is `parseInt`, so I belive it cannot be `Nan`.

Reply:
fixed